### PR TITLE
Fix monaco editor widget error

### PR DESCRIPTION
**Pull request by spyraklas**

### CODE REVIEW

**Comment by spyraklas** [comment]
@dotnet-policy-service agree

**Comment by sebastienros** [comment]
Great to see you succeeded, and congrats on your first PR

image

File: src/OrchardCore.Modules/OrchardCore.ContentFields/Views/HtmlField-Monaco.Edit.cshtml [hunk 1]
```diff
@@ -21,65 +21,71 @@
...
-    document.addEventListener('DOMContentLoaded', function() {
+    document.addEventListener('DOMContentLoaded', function () {
         require(['vs/editor/editor.main'], function () {
 
-            var settings = @Html.Raw(monacoSettings.Options);
-            if (settings.automaticLayout == undefined) {
-                settings.automaticLayout = true;
-            }
-            settings.language = 'html';
+            var divEditor = document.getElementById('@Html.IdFor(x => x.Html)_editor');
```
Comment (MikeAlhayek) [inline src/OrchardCore.Modules/OrchardCore.ContentFields/Views/HtmlField-Monaco.Edit.cshtml:27]: you can change `var` to `const` and eliminate the `if (divEditor != undefined || divEditor != null) {` check.


or if you don't want to use `const`, then do this instead

```
if (!divEditor) {
    return;
}
```

which should give you the same result but much less indentation.

**Review by MikeAlhayek — COMMENTED** [review]
Did you confirm that this work for the following scenarios? 

1) content item with no widgets, add at least 2 widgets using the UI.
2) load content item with existing "saved" widgets, does existing widgets sill work? Does adding new widgets wrok?

**Comment by MikeAlhayek** [comment]
Thank Sebestian for addressing my feedback,
@spyraklas Good job on your first contribution! Your PR will soon be merged.

**Pull request merged**